src/main/java/uk/gov/ch/developer/docs/controller/developer/OAuthToken.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.ch.developer.docs.controller.developer;
+
+public class OAuthToken {
+
+    private String expires_in;
+    private String token_type;
+    private String refresh_token;
+    private String access_token;
+
+    private OAuthToken() { }
+
+    public String getExpires_in() {
+        return expires_in;
+    }
+
+    public void setExpires_in(String expires_in) {
+        this.expires_in = expires_in;
+    }
+
+    public String getToken_type() {
+        return token_type;
+    }
+
+    public void setToken_type(String token_type) {
+        this.token_type = token_type;
+    }
+
+    public String getRefresh_token() {
+        return refresh_token;
+    }
+
+    public void setRefresh_token(String refresh_token) {
+        this.refresh_token = refresh_token;
+    }
+
+    public String getAccess_token() {
```

Comment:
Please name member using Java styling names, i.e. camel case. If they need to be mapped to JSON, then use attributes.

Reply:
I think OAuth token might be better placed in the ouath package.

Reply:
Done